ui/component/comment/view.jsx

```diff
@@ -1,73 +1,206 @@
 // @flow
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { isEmpty } from 'util/object';
 import relativeDate from 'tiny-relative-date';
 import Button from 'component/button';
 import Expandable from 'component/expandable';
 import MarkdownPreview from 'component/common/markdown-preview';
 import ChannelThumbnail from 'component/channelThumbnail';
+import { Menu, MenuList, MenuButton, MenuItem } from '@reach/menu-button';
+import Icon from 'component/common/icon';
+import * as ICONS from 'constants/icons';
+import { FormField, Form } from 'component/common/form';
+import CommentReply from '../commentReply/index';
+import classnames from 'classnames';
 
 type Props = {
-  author: string,
-  authorUri: string,
-  message: string,
-  timePosted: number,
-  claim: ?Claim,
+  uri: string,
+  author: ?string, // LBRY Channel Name, e.g. @channel
```

Comment:
These changes were done in a previous PR, they already exist on master
https://github.com/lbryio/lbry-desktop/blob/master/ui/component/comment/view.jsx#L15

Reply:
Alright let me know if I still need to do something, I rebased with master

Reply:
Looks good! I'll give this a review/test later today.